docs/shared_parsers_catalog/dnf_modules.srt

```diff
@@ -0,0 +1,3 @@
+.. automodule:: insights.parsers.dnf_modules
```

Comment:
@tkasparek this file is named incorrectly, it should be `.rst` instead of `.srt`.  Once you rename it please check your documentation as described here to make sure that it renders correctly.  https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-docstring

Reply:
Should be resolved now.